src/scenes/Question/scenes/Edit/components/Tips/index.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react'
+
+import { Card, CardText } from 'react-toolbox/lib/card'
+import { List, ListItem, ListDivider } from 'react-toolbox/lib/list'
+import FontIcon from 'react-toolbox/lib/font_icon'
+
+import tips from './tips.json'
+
+class Tips extends Component {
+  render () {
+    return (
+      <Card {...this.props}>
+        <CardText>
+          <h3 style={{ textAlign: 'center' }}>
+            Tips to write great questions on the FAQ
+          </h3>
+          <List selectable>
+            {tips.good.map((tip, i) => (
+              <ListItem
```

Comment:
Is it possible to make a reusable component ? There seems to be some duplicated code...

Reply:
Are you talking about the maps ?

Reply:
The ListItem

Reply:
Done 👌